lib/pages/survey/survey_page.dart

```diff
@@ -1,92 +1,108 @@
+import 'dart:ui';
+
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:get/get.dart';
-import 'package:survey/navigator/const.dart';
 import 'package:survey/pages/survey/survey_controller.dart';
 
 class SurveyPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    final surveyId = Get.arguments[DATA_SURVEY_ID];
-
-    // TODO: work on the Integrated data later
     return Scaffold(
       body: GetBuilder(
         init: SurveyController(),
-        builder: (controller) => Stack(
-          children: [
-            Container(
-              decoration: BoxDecoration(
-                image: DecorationImage(
-                  image: NetworkImage(
-                      "https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_l"),
-                  fit: BoxFit.cover,
-                  colorFilter: ColorFilter.mode(
-                      Colors.black.withOpacity(0.5), BlendMode.overlay),
+        builder: (controller) => GetX<SurveyController>(
+          builder: (state) {
+            return state.optionalSurvey.isPresent
+                ? SurveyDetail(state: state)
```

Comment:
I think putting `survey` is enough. How do you think? 😄 
```suggestion
                ? SurveyDetail(survey: state.optionalSurvey.value)
```

Reply:
oh yes 👌